### DISCUSSION TITLE
Welcome to mLRS Discussions!

### DISCUSSION THREAD

**Discussion by olliw42** [olliw42/mLRS#59] [Announcements]
## 👋 Welcome!

We are using Discussions for in-depth discussions on development topics. It is not supposed to be used as support channel or for general, non-dev questions. For these purposes please consider joining the rcgroups thread or the Discord server.

**Comment by nithinteja1** [comment]
Hi are there any plans of mLRS for radiomaster nomad series ? .

**Comment by olliw42** [comment]
if some takes on the effort it will happen, else not
:)
